features-json/av1.json

```diff
@@ -590,7 +590,7 @@
     "1":"Can be enabled in Edge on Windows 10 or later by installing the [AV1 Video Extension from the Microsoft Store](https://www.microsoft.com/store/productId/9MVZQVXJBQ9V)",
     "2":"Firefox 65 only supported AV1 on Windows 64-bit",
     "3":"Firefox 66 only supported AV1 on Windows and macOS",
-    "4":"Supported only for devices with hardware decoder i.e. A17 Pro (in e.g. iPhone 15 _Pro_ (Max)) or later"
+    "4":"Supported only for devices with hardware decoder, i.e. A17 Pro (e.g. iPhone 15 Pro) or M3 chip or later"
```

Comment:
I'd suggest something like this instead: 

Supported on devices with hardware decoder, including iPhone 15 Pro, M3 iMac, M3 MacBook Pro, etc.

Reply:
Adjusted!